src/Go.CodeAnalysis/Lex/Operators.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Go.CodeAnalysis.Lex
+{
+    public static class Operators
```

Comment:
These don't appear to be used anywhere. The reason I didn't make these constants is that there's overhead to allocating and indexing into a string, so I've used char constants.

Reply:
I see. Maybe eventually we should use a regex based pattern matching? Maintaining the switch case is a headache.